javascript/javascript1/week2/Candy Helper/candyhelper.js

```diff
@@ -0,0 +1,35 @@
+const boughtCandyPrices = [];
+const amountToSpend = Math.random() * 100;
+
+function addCandy(candyType, weight) {
+  const candyPriceTable = {
```

Comment:
very nice! You can try and think what will happen if `candyType` argument is of some weird value that is not in this object - what should the funcion do then and what does it do now? Hint: you can check `Object.keys()`